src/pages/CategoriesPage/AddCategory/AddNewCategoryComponent.tsx

```diff
@@ -7,73 +7,86 @@ import {
 } from '@mui/material';
 import TextField from '@mui/material/TextField';
 import { Formik } from 'formik';
-import { CreateCategoryModel } from '../../../models/categories/CreateCategoryModel';
-import { createValidationSchema } from '../../../validationSchemas/createValidationSchema';
+import { CreateUpdateCategoryModel } from '../../../models/categories/CreateCategoryModel';
+import { validationSchema } from '../../../validationSchemas/validationSchema';
 import { styles } from '../styles';
 
 interface Props {
   createModalOpen: boolean;
-  onSubmit: (values: CreateCategoryModel) => void;
+  onSubmit: (values: CreateUpdateCategoryModel) => void;
   onToggleCreateStatusClick: () => void;
 }
 
 const AddNewCategoryComponent = (props: Props) => {
   return (
     <Dialog open={props.createModalOpen}>
-      <DialogTitle sx={styles.dialogTitle}>CREATE CATEGORY</DialogTitle>
+      <div style={styles.dialogTitleWrapper}>
```

Comment:
Do not use DIV. Use Box instead.

Reply:
All divs in the modal windows have been replaced by the Box components